koocook_core/tests/test_support/test_quantity.py

```diff
@@ -74,4 +82,19 @@ def test_parse_quantity_is_inverse_of_get_db_str(self):
                         except ValueError as e:
                             raise self.failureException(f'test raised {e.__class__.__name__} unexpectedly') from e
 
-    # TODO: Improve coverage in quantity.py
+    def test_add_with_both_amount_are_fraction(self):
+        self.setUp()
+        summation1 = self.quantity1 + self.quantity1
+        self.assertEqual(summation1, self.quantity2)
+        summation2 = self.quantity1 + self.quantity2
+        self.assertEqual(summation2, self.quantity4)
+
+    def test_add_with_both_amount_are_int(self):
+        self.setUp()
+        summation = self.quantity3 + self.quantity4
+        self.assertEqual(summation, self.quantity5)
+
+    def test_add_fraction_with_int(self):
+        self.setUp()
+        summation = self.quantity1 + self.quantity4
+        self.assertEqual(summation, self.quantity6)
```

Comment:
This is not how you normally use `TestCase.setUp()`. Why call it again every time?